src/charts/donut/examples/donut-pct-chart.js

```diff
@@ -26,6 +26,15 @@
  * <li>.tooltipFn(d)   - user defined function to customize the tool tip (optional)
  * <li>.centerLabelFn  - user defined function to customize the text of the center label (optional)
  * <li>.onClickFn(d,i) - user defined function to handle when donut arc is clicked upon.
+ * <li>.label          - object containing properties for external label (optional)
```

Comment:
Took using ```side-label``` into consideration, but when the label can be used on the right, left, and bottom it becomes a slight misnomer, so changed it to ```label```. 

Was also going to create it as a separate attribute similar to ```center-label```, but then we blend the use of how ```center-label``` functions. We have both a configuration aspect, which takes a function, and the attribute, which takes a string. Based on that took the configuration route all in one, instead of a blended approach, which for myself felt confusing. 

I do agree the ```label``` label should probably be changed to something else, I like ```labelConfig```, had also thought maybe ```outsideLabel```, but the ```Config``` suffix seems more appropriate. 

As a quick fix, I can go ahead and change the label to ```labelConfig``` while we continue to discuss.